addons/arsenal/stringtable.xml

```diff
@@ -433,5 +433,17 @@
             <Italian>Impossibile aprire l'arsenale ACE</Italian>
             <Japanese>ACE 武器庫を開けません</Japanese>
         </Key>
+        <Key ID= "STR_ACE_Arsenal_portLoadoutsText">
+            <English>import BI VA loadouts to ACE Arsenal</English>
```

Comment:
`Import` (capitalization).

I think something like `Import Vanilla Arsenal loadouts to ACE Arsenal` would be more understandable, but longer.

Reply:
The issue is that it translates poorly into other languages, on the other hand "BI VA" is pretty much universal.